code/game/machinery/ammolathe.dm

```diff
@@ -52,21 +57,23 @@
 		new /obj/item/ammo_storage/magazine/c45/empty(), \
 		new /obj/item/ammo_storage/magazine/uzi45/empty(), \
 		new /obj/item/ammo_storage/magazine/a50/empty(), \
-		new /obj/item/ammo_storage/magazine/a75/empty(), \
-		new /obj/item/ammo_storage/magazine/a762/empty(), \
 		new /obj/item/ammo_storage/magazine/a12ga/empty(), \
-		new /obj/item/ammo_storage/magazine/a12mm/empty(), \
 		),
 		"Misc_Other"=list(
 		new /obj/item/ammo_storage/speedloader/c38/empty(), \
 		new /obj/item/ammo_storage/speedloader/a357/empty(), \
-		new /obj/item/ammo_storage/speedloader/a762x55/empty(), \
 		new /obj/item/ammo_storage/speedloader/shotgun(), \
 		),
 		"Hidden_Items" = list(
-		new /obj/item/weapon/reagent_containers/glass/beaker/vial(), \
-		new /obj/item/weapon/reagent_containers/syringe(), \
-		) //Syringes and vials are technically an ammo.
+		new /obj/item/ammo_storage/speedloader/a762x55/empty(), \
```

Comment:
To here? Mags should remain in the mag section. You're making it unorganized by adding things to categories they shouldn't be in. Hacked content wasn't about dangerous stuff being locked behind it. It was ment for items that are only technically ammo.
Move them to their proper categories.

Reply:
Contraband content on the autolathe is specifically for dangerous stuff, so I don't see why the ammolathe shouldn't have the same deal. Hacking the ammolathe to unlock vials and syringes is just ridicilous; in my opinion it makes far more sense for contraband items on it to be "sketchy" ammunition for guns sec probably shouldn't even have.

Reply:
That's some dumb reasoning EVERYTHING in the ammolathe is dangerous by definition of them being guns.